rpc/src/rpc_actor.rs

```diff
@@ -134,6 +141,11 @@ impl Receive<ShellChannelMsg> for RpcServer {
 
     fn receive(&mut self, _ctx: &Context<Self::Msg>, msg: ShellChannelMsg, _sender: Sender) {
         if let ShellChannelMsg::NewCurrentHead(_, block) = msg {
+            // prepare main chain_id
+            let chain_id = parse_chain_id(MAIN_CHAIN_ID, &self.env).unwrap();
+            // warm-up - calls where chain_id + block_hash
+            let _ = crate::services::base_services::get_block_metadata(&chain_id, &block.hash,&self.env);
```

Comment:
@mambisi 
we need to add here all cached calls,

later we could do those precache calls in parallel with `try_join!` or something

Reply:
Ok